format/spec.md

```diff
@@ -1370,3 +1370,16 @@ Writing v2 metadata:
     * `sort_columns` was removed
 
 Note that these requirements apply when writing data to a v2 table. Tables that are upgraded from v1 may contain metadata that does not follow these requirements. Implementations should remain backward-compatible with v1 metadata requirements.
+
+## Appendix F: Implementation Notes
+
+This section covers topics not required by the specification but recommendations for systems implementing the Iceberg specification
+to help maintain a uniform experience.
+
+### Point in Time Reads (Time Travel)
+
+Iceberg supports two types of histories for tables. A history of previous "current snapshots" stored in ["snapshot-log" table metadata](#table-metadata-fields) and [parent-child lineage stored in "snapshots"](#table-metadata-fields). These two histories 
+might indicate different snapshot IDs for a specific timestamp. The discrepancies can be caused by a variety of table operations (e.g. updating the `current-snapshot-id` of the table).
+
+When processing point in time queries the Iceberg community has chosen to use "snapshot-log" metadata to lookup the table state
```

Comment:
> has chosen to use "snapshot-log" metadata to lookup the table state

But isn't it an optional field in the spec? 
https://github.com/apache/iceberg/pull/8982

What if the implementations of spec doesn't maintain the "snapshot-log" history?

Reply:
If you don't have this information, then time travel to a specific time would fail because you don't know what the state of the table was at that time. It should definitely not change to use the parent/child relationship because the time travel behavior could be inconsistent depending on how the log is maintained.

Reply:
Ack.

Reply:
Added a sentence stating an informative error message should be raised.